libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -99,9 +99,9 @@ export interface ISimulatorContext {
   readonly serverUrl: string;
 
   /**
-   * Find a resource simulation by its handle. Throws if the handle isn't valid.
+   * Obtain a client given a resource's handle.
    */
-  findInstance(handle: string): ISimulatorResourceInstance;
+  getClient(handle: string): any;
```

Comment:
Was `ISimulatorResourceInstance` inaccurate? If so, maybe we can use `unknown` here to avoid accidental use as `any`.

Reply:
I think so, yeah, since the interface currently models the simulator's lifecycle methods (init, cleanup, save) and we don't really want clients to be able to call those anyways. `unknown` sounds ok to me